SwiftWisdom/Core/UIKit/UIView+Extensions.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIView+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 12/29/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+
+    /// Zooms a view by scaling using the provided factor.
+    func ip_zoom(by factor: CGFloat) {
+        transform = CGAffineTransform(scaleX: factor, y: factor)
```

Comment:
Might be helpful to have one that uses the original transform.

e.g. 
```
view.ip_zoomFromCurrentBy(2)
view.ip_zoomFromCurrentBy(2)
```
Would actually increase the zoom by a factor of 4